### ISSUE TITLE
Unable to unpublish linter-js-standard-tenx

### ISSUE THREAD

**Issue by jeremyosborne** [atom/apm#541]
tl;dr I've seen others here ask @thedaniel to unpublish their package. If this is correct, would you unpublish and delete all versions of this: https://atom.io/packages/linter-js-standard-tenx

Last night I unknowingly ran into the problem that I've seen reported elsewhere: fork a repo, publish a package, delete / recreate the repo to break the fork (as I'm digressing from the original package too much), and could not unpublish my package.

I apologize for being messy and going ahead and recreating the package under a different name. I'm new to `apm` (and quite enjoying it) but wanted to move on and give my team a package to work with.

**Comment by Alhadis** [comment]
I think this is a bug with APM. :| I'm unable to unpublish my repo either! See #538.

**Comment by jhabdas** [comment]
@thedaniel do you know if it is possible to unpublish or rename this package until Atom lists packages by popularity?

**Comment by jhabdas** [comment]
Thanks @50Wliu!

**Comment by jasonrudolph** [comment]
This was a bug in apm's backend API. I think we've got it resolved now. 🤞

@jeremyosborne: If you still want to unpublish the package, can you please try unpublishing again? If you've already run `apm login`, then this command should do the trick:

    apm unpublish linter-js-standard-tenx

If you still run into trouble unpublishing, please let us know.

**Comment by jeremyosborne** [comment]
Thanks! It worked. Should be unpublished.

**Comment by jhabdas** [comment]
@jeremyosborne thank you very much, kind sir.
